scripts/game.js

```diff
@@ -42,6 +26,46 @@ class Game {
     return { elapsedSeconds, elapsedMinutes };
   }
 
+  static stringSearch({ string, keyword }) {
+    if (string.search(keyword) !== -1) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
You can omit the else here, since you return anyways.

Reply:
True, but does it return false? It seems weird to not return false if it indeed is false?

Reply:
I meant you can write it like this:

```
if (string.search(keyword) !== -1) {
  return true;
}

return false;
```

I just now realised you can even write it like this 😆 

```javascript
return string.search(keyword) !== -1;
```

Reply:
Oh yeah I get what you mean. Yes ofcourse, I will fix that!